Documentation/Bluetooth.md

```diff
@@ -0,0 +1,99 @@
+# Bluetooth Low Energy
+
+The Pinecilv2 has hardware support for Bluetooth Low Energy (BLE). This protocol allows reading and writing of parameters to the Pinecil during runtime.
+
+The BLE interface advertises three services, these provide access to Live telemetry as well as the ability to read/write settings.
+These are outlined in more detail below.
+
+Pinecil devices advertise themselves on BLE as `Pinecil-XXXXXXX`.
+They also include the UUID `9eae1000-9d0d-48c5-AA55-33e27f9bc533` in the advertisement packet to allow for filtering.
+
+Unless otherwise noted, all data is sent and received as Little-Endian.
+
+As of the time of writing this, notifications are not fully implemented so data will need to be polled. Notification/Indication support will come when there is time to implement it.
+
+## Using the BLE Interface
+
+It is advised to follow the below points when first implementing a BLE integration. Of course once the integration is working feel free to deviate from these. These are just _suggested_ ideas to help kickstart.
+
+1. When filtering for devices, its preferable to filter on the UUID `9eae1000-9d0d-48c5-AA55-33e27f9bc533` than device name if possible.
```

Comment:
```suggestion
1. When filtering for devices, its preferable to filter by the UUID `9eae1000-9d0d-48c5-AA55-33e27f9bc533`, rather than by the device name if possible.
```